testing/ostest/semtimed.c

```diff
@@ -77,7 +77,11 @@ static void ostest_gettime(struct timespec *tp)
       printf("ostest_gettime: ERROR: clock_gettime failed\n");
       ASSERT(false);
     }
+#ifdef CONFIG_SYSTEM_TIME64
```

Comment:
I do not fully understand this ifdef. We have
```
#ifdef CONFIG_SYSTEM_TIME64
typedef uint64_t     clock_t;
typedef uint64_t     time_t;         /* Holds time in seconds */
#else
typedef uint32_t     clock_t;
typedef uint32_t     time_t;         /* Holds time in seconds */
#endif
```
so `tv_sec` is unsigned always and not based on `SYSTEM_TIME64`. Why `tp->tv_sec < 0` is still checked if `CONFIG_SYSTEM_TIME64` is defined?